packages/example-getting-started/docs/8-putting-it-together.md

```diff
@@ -1,61 +1,84 @@
 ### Putting it all together
 
-Now that we've got all of our artifacts made, let's set them up in our
-application!
+We've got all of our artifacts now, and all that's left is to bind them to our
+[Application](http://loopback.io/doc/en/lb4/Application.html) so that LoopBack's
+[Dependency injection](http://loopback.io/doc/en/lb4/Dependency-injection.html)
+system can tie it all together for us!
 
-We'll define a new helper function for setting up the repositories, as well
-as adding in our new controller binding.
+LoopBack's
+[boot module](https://github.com/strongloop/loopback-next/tree/master/packages/boot)
+will automatically discover our controllers, repositories, datasources and
+other artifacts and inject them into our application for use.
+In the current example, we've manually loaded our repository so that it can be
```

Comment:
repositories are also bootable now. I'm not sure of the point of mentioning mocking / stubbing for unit testing purposes here ...

Reply:
At the time I wrote this, that wasn't the case!